_data/sites.json

```diff
@@ -1088,6 +1088,17 @@
             "https://www.bet365.com"
         ]
     },
+	
+	{
+        "name": "Bethesda",
+        "url": "https://help.bethesda.net/app/incident10/?prod=711&cat=1220",
+        "difficulty": "hard",
+        "notes": "To delete the account, login first on Bethesda.net, then click on the above link and select Bethesda as the product (note that the page takes a while to load), then select 'Delete Personal Information and Account' and fill out the form, this will send a prefilled support contact form for account deletion. You may also send an email to privacy@support.zenimax.com ",
+        "email": "privacy@support.zenimax.com",
+        "domains": [
+            "bethesda.net"
+        ]
+    },
```

Comment:
```suggestion

    {
        "name": "Bethesda",
        "url": "https://help.bethesda.net/app/incident10/?prod=711&cat=1220",
        "difficulty": "hard",
        "notes": "To delete the account, login first on Bethesda.net, then click on the above link and select Bethesda as the product (note that the page takes a while to load), then select 'Delete Personal Information and Account' and fill out the form, this will send a prefilled support contact form for account deletion. You may also send an email to privacy@support.zenimax.com ",
        "email": "privacy@support.zenimax.com",
        "domains": [
        "bethesda.net"
        ]
    },
```